apps/mobile/src/features/activity/activity-item.tsx

```diff
@@ -0,0 +1,66 @@
+import { memo, useCallback } from 'react';
+
+import { Balance } from '@/components/balance/balance';
+import type { ResponsiveValue } from '@shopify/restyle';
+
+import { type ActivityView } from '@leather.io/features';
+import { ActivityAvatarIcon, Cell, type Theme } from '@leather.io/ui/native';
+
+import { useOpenURL } from '../browser/browser/use-open-url';
+
+interface ActivityItemProps {
+  item: ActivityView;
+}
+
+function ActivityItemComponent({ item }: ActivityItemProps) {
+  const { balances, title, caption, activityLink } = item;
+  const { openURL } = useOpenURL();
```

Comment:
```suggestion
  const { openUrl } = useOpenUrl();
```

https://www.notion.so/trustmachines/Naming-Camelcase-acronyms-119d74b694c78080974aee5e3bc18bc9?source=copy_link

Reply:
I'll update this now but doing a quick search on `dev` and I get 55 results for `openURL` so it's not a new thing

Reply:
It seems this name comes from the `react-native` `Linking` package. I've refactored and renamed our own uses to match this convention